sdk/scheduler/src/main/java/com/mesosphere/sdk/dcos/Capabilities.java

```diff
@@ -37,6 +56,10 @@ public boolean supportsCNINetworking() throws IOException {
         return hasOrExceedsVersion(1, 9);
     }
 
+    public boolean supportsPreReservedResources() {
+        return false;
```

Comment:
Mesos doesn't have a way expose capabilities.  I'll be based on version number.  Will update this when they PRs in DC/OS and Mesos are merged.